formulas/dnsmasq/files/make-resolv-conf

```diff
@@ -0,0 +1,9 @@
+# Adds a check to dhclient. If disk usage reaches 96% or greater, the function
+# that overwrites /etc/resolv.conf will do nothing. This prevents
+# /etc/resolv.conf from being partially overwritten, which occurs when the disk
+# is full.
+if df -h / | grep -qoP "(9[6-9]|100)(?=%)"; then
```

Comment:
Also I would use the full path, df will resolve to the mount point and tell you the free space there. Though granted it seems unlikely for someone to mount /etc different from /.

I'm somewhat concerned that if the disk is say 98% full the script will run and returns 0 but not actually update the config file. This could lead to confusing situations.

It may be better to create a new resolv.conf.tmp file based on the old one and then replace the old with new, that way it will it either work even if there's only a couple of KB left to work with.